tipg/dbmodel.py

```diff
@@ -71,14 +140,17 @@ class Parameter(Column):
     default: Optional[str] = None
 
 
-class Table(BaseModel):
-    """Model for DB Table."""
+class Collection(BaseModel):
```

Comment:
Next add `from_file` class method to create a Collection from a SQL file

https://github.com/developmentseed/timvt/blob/master/timvt/layer.py#L252-L258